src/main/java/edu/cornell/kfs/fp/CuFPConstants.java

```diff
@@ -134,6 +134,7 @@ public static ScheduleTypes fromName(String name) {
         public static final String RECURRING_DV_DOCUMENT_TYPE_NAME = "RCDV";
         public static final String RECURRING_DV_EXPLANATION_TO_DV_NOTE_STARTER = "DV created by recurring DV: ";
         public static final String RECURRING_DETAILS_TAB_NAME = "RecurringDetails";
+        public static final String RECURRING_DV_CANCEL_PARMENTS_PERMISSION_NAME = "CANCEL RECURRING DISBURSEMENT VOUCHER PAYMENTS";
```

Comment:
Minor typo in variable name, should be "..._PAYMENTS_..." instead of "...PARMENTS..."